traffic_portal/app/src/common/api/AuthService.js

```diff
@@ -21,56 +21,48 @@ var AuthService = function($rootScope, $http, $state, $location, $q, $state, htt
 
     this.login = function(username, password) {
         userModel.resetUser();
-        return httpService.post(ENV.api['root'] + 'user/login', { u: username, p: password })
-            .then(
-                function(result) {
-                    $rootScope.$broadcast('authService::login');
-                    var redirect = decodeURIComponent($location.search().redirect);
-                    if (redirect !== 'undefined') {
-                        $location.search('redirect', null); // remove the redirect query param
-                        $location.url(redirect);
-                    } else {
-                        $location.url('/');
-                    }
-                },
-                function(fault) {
-                    // do nothing
+        return httpService.post(ENV.api['root'] + 'user/login', { u: username, p: password }).then(
+            function(result) {
+                $rootScope.$broadcast('authService::login');
+                const redirect = decodeURIComponent($location.search().redirect);
+                if (redirect !== 'undefined') {
+                    $location.search('redirect', null); // remove the redirect query param
+                    $location.url(redirect);
+                } else {
+                    $location.url('/');
                 }
-            );
+            },
+            function(err) {
+               console.error(err);
+            }
+        );
     };
 
     this.tokenLogin = function(token) {
```

Comment:
i would just leave the tokenLogin function alone unless your sure this doesn't change behavior. seems like it would.

Reply:
actually,  nm. i think this is fine